data/json/hobbies.json

```diff
@@ -397,21 +598,51 @@
     "skills": [ { "level": 2, "name": "melee" }, { "level": 3, "name": "unarmed" }, { "level": 3, "name": "dodge" } ],
     "traits": [ "MARTIAL_ARTS3" ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "blackbelt_hobby",
+    "name": "Black Belt",
+    "description": "You've been doing martial arts for years and have earned the coveted black belt in a martial art of your choice.",
+    "points": 10,
+    "skills": [
+      { "level": 5, "name": "melee" },
+      { "level": 5, "name": "unarmed" },
+      { "level": 5, "name": "dodge" },
+      { "level": 3, "name": "swimming" }
+    ],
+    "traits": [ "PROF_MA_BLACK" ]
+  },
   {
     "type": "profession",
     "subtype": "hobby",
     "id": "melee_training",
-    "name": "Melee Training",
+    "name": "Melee Training (Intermediate)",
     "description": "You have practiced fighting with weapons.  You start with your choice of Barbaran Montante, Bōjutsu, Eskrima, Fencing, Fior Di Battaglia, Medieval Swordsmanship, Niten Ichi-Ryu, Pencak Silat, or Sōjutsu.",
-    "points": 6,
+    "points": 7,
     "skills": [
-      { "level": 2, "name": "melee" },
+      { "level": 3, "name": "melee" },
       { "level": 3, "name": "bashing" },
       { "level": 3, "name": "stabbing" },
       { "level": 3, "name": "cutting" }
     ],
     "traits": [ "MARTIAL_ARTS5" ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "melee_training_expert",
+    "name": "Melee Training (Expert)",
+    "description": "Despite firearms making traditional melee weapons largely obsolete, you have extensive experience fighting in hand-to-hand combat with tools of war.  Maybe it was full-contact HEMA or SCA sparring, or maybe you're just a badass commando with a machete.  You start with your choice of Barbaran Montante, Bōjutsu, Eskrima, Fencing, Fior Di Battaglia, Medieval Swordsmanship, Niten Ichi-Ryu, Pencak Silat, or Sōjutsu.",
+    "points": 14,
```

Comment:
This is a whole lot of points when you aren't even getting any proficiency. You are getting a martial arts. I feel like 12 points should be enough unless you add a weapon. The description seems to suggest a machete.